game/templates/game/level_moderation.html

```diff
@@ -110,12 +110,13 @@
 
 {% if levelData %}
   <div class="tableWrapper">
-    {% if thead %}
     <table id="" class="CSSTable wide alternate smaller">
       <tr>
-      {% for header in thead %}
-        <th> {{header}} </th>
-      {% endfor %}
+        <th>{% trans "Student" %}</th>
+        <th>{% trans "Level name" %}</th>
+        <th>{% trans "Shared with" %}</th>
+        <th>{% trans "Play" %}</th>
+        <th>{% trans "Delete" %}</th>
```

Comment:
For this one, I'd rather go with Mike's suggestion, also because I would keep the list in Python view in case we need it for something else or we need to add elements - so I'd add the ugettext around them but the trans around scoreboard is fine I think.